utilities/fill_counties_api.py

```diff
@@ -0,0 +1,119 @@
+import csv
```

Comment:
this file looks like a boilerplate of fill_counties.py ... couldn't you reuse some codes from there?

Reply:
Thanks for review @xmnlab, We have another ‘app’ that depends on models.py save () method, so I passed this save () method into the fill_counties_api.py app and kept the other version of fill_counties and models by reading the data obtained from br-atlas. Perhaps a new issue to unify fill_counties, fill_states, fill_stations and update_mun_w_pop.py using the same save () method.

Reply:
for example to_row, it is basically the same for both ... boilerplates are very hard to maintain